cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -24,6 +25,22 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         response = event["Records"][0]["cf"]["response"]
 
+        try:
+            want_digest = request["headers"]["want-digest"][0]["value"]
+        except KeyError:
+            LOG.debug(
+                "Could not read want-digest from response", exc_info=True
```

Comment:
Request, not response.

```suggestion
                "Could not read want-digest from request", exc_info=True
```